fw/http_frame.c

```diff
@@ -1002,7 +1002,7 @@ tfw_h2_increment_wnd_sz(long int *window, unsigned int wnd_incr)
 	 * is sent.
 	 */
 	if (new_window > MAX_WND_SIZE)
-		return T_DROP;
+		return T_BLOCK;
```

Comment:
Previously we also send  TCP RST because T_DROP previously was same as T_BLOCK now! In this commit i just implement new error code SS_BLOCK and change T_DROP to T_BLOCK, because now T_BLOCK has same behavior as it previously was to T_DROP!

Reply:
Yes, I see that the new `T_BLOCK` is essentially the previous `T_DROP` and you basically don't change anything with this patch. However, the whole TCP RST logic is relatively new and the previous `T_DROP` also didn't fit well to the new different behaviors with RST or FIN. The point is that now we should revise where do we return `T_BLOCK` and where `T_DROP` - previously this wasn't important, but now it is.

However, I propose just to create a new task for this (see another my comment) and just comment this code and maybe some other places with TODO and the issue number.